README.rst

```diff
@@ -97,9 +97,8 @@ This project depends on
 
 1. `Spyder <https://github.com/spyder-ide/spyder>`_
 2. `Tornado <https://github.com/tornadoweb/tornado>`_
-3. `Pexpect <https://github.com/pexpect/pexpect>`_
-4. `Coloredlogs <https://github.com/xolox/python-coloredlogs>`_
-5. `xterm.js <https://github.com/sourcelair/xterm.js>`_
+3. `Coloredlogs <https://github.com/xolox/python-coloredlogs>`_
+4. `xterm.js <https://github.com/sourcelair/xterm.js>`_
```

Comment:
Please change this:

1. Mention that `pexpect` is needed on Linux and macOS.
2. Add `pywinpty` here, saying also that's only needed for Windows.